metis-indexing/src/main/java/eu/europeana/indexing/mongo/property/MongoObjectUpdater.java

```diff
@@ -6,20 +6,21 @@
 /**
  * Updater of mongo objects.
  *
- * @param <R> The type of the record to update.
+ * @param <R> The type of the object to update.
  * @param <A> The type of the ancestor information (information from parents).
  */
 public interface MongoObjectUpdater<R, A> {
```

Comment:
Why not just add a default method here for the delete that does nothing and it's only implemented on the classes that needed, in this case WebResourceMetaInfoUpdater?

Reply:
Here are my reasons:
1. Not all objects can ever be removed (we already have removal code for whole records).
2. It would go against consistency. We would not call those remove methods for all subclasses. This specific one is called from code that deals with metadata meta info only, and I think it would be overkill to implement object removal everywhere just on the chance that someone later implements a functional removal method. It follows that it would create the false impression that by actually implementing the remove method, we could enable removal. In summary: there is a lack of consistency in that situation.
3. I also wanted to keep the change relatively minimal (small). Perhaps this may be a hotfix.
Does this make sense? Or would you still prefer to merge the two interfaces?

Reply:
But you don't have to implement it anywhere, that's the idea of the default method on the interface.
Did you mean implementing it on all subclasses?